### ISSUE TITLE
angular codesandbox in the documentation is broken 

### ISSUE THREAD

**Issue by mhamri** [nolimits4web/swiper#5681]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/s/swiper-angular-example-9edj9

### Bug description

if you click on this sandbox 
![image](https://user-images.githubusercontent.com/8991783/165594628-5fb71134-babc-4105-ae32-e2ec4ab7321d.png)

you will see this error 

![image](https://user-images.githubusercontent.com/8991783/165594223-b7e59be6-8cc8-4254-b03a-0d7ef2e6ce12.png)

if I dig up the creator sandbox I could find another sandbox that supposed to work 
https://codesandbox.io/s/swiper-angular-example-forked-hb8ypr?file=/package.json

but 

![image](https://user-images.githubusercontent.com/8991783/165594898-880c505a-b963-4b0b-90aa-ab965c7785ad.png)

so basically there is no working sandbox in the angular documentation 

### Expected Behavior

view a working sandbox 

### Actual Behavior

all sandboxes are broken 

### Swiper version

^8.0.6

### Platform/Target and Browser Versions

i've tested sandbox on firefox and chrome

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR
